sym.go

```diff
@@ -0,0 +1,88 @@
+package fuego
+
+import (
+	"reflect"
+	"strconv"
+)
+
+type FuegoKind int
```

Comment:
type name will be used as fuego.FuegoKind by other packages, and that stutters; consider calling this Kind